.github/workflows/snapshot-service-image.yml

```diff
@@ -0,0 +1,57 @@
+name: Snapshot Service Image
+
+# Cancel workflow if there is a new change to the branch.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+on:
+  push:
+    branches: [main]
+  merge_group:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build-and-push-docker-image:
+    name: Build images and push to GHCR
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: List cached docker images
+        run: docker image ls
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Login to Github Packages
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # This step yields the following labels:
+      # - date+sha, e.g. 2023-01-19-da4692d,
+      # - latest,
+      - name: Docker Meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/chainsafe/forest-snapshot-service
+          tags: |
+            type=raw,value={{date 'YYYY-MM-DD'}}-{{sha}}
+            type=raw,value=latest,enable={{is_default_branch}}
+
+      - name: Build image and push to GitHub Container Registry
+        uses: docker/build-push-action@v5
+        with:
+          context: ./images/snapshot-service/
```

Comment:
```suggestion
          context: ./images/snapshot-service/
          load: true
```

Reply:
I believe without `load: true`, `docker image ls` steps won't list the output image

Reply:
Nvm it actually does. https://github.com/ChainSafe/forest-iac/actions/runs/7726607692/job/21063223556?pr=400#step:7:30